tests/integration/test_storage_kafka/test.py

```diff
@@ -574,39 +574,13 @@ def test_kafka_formats(kafka_cluster):
+            ],
```

Comment:
I guess with empty messages it will still break, but it's acceptable.

Reply:
yes, agree it, empty message size is zero also.